### PR TITLE
Cypress test. Canvas 3D. Create a cuboid.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Cypress test to check a cuboud creation. Based on visual regression testing.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@ActiveChooN 

Please, look at this patch and give your opinion.

**Comment by dvkruchinin** [comment]
Set the WIP status just in order to implement this patch after #3003

File: tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const_canvas3d';
+
+context('Canvas 3D functionality. Add cuboid.', () => {
+    const caseId = '64';
+
+    const screenshotsPath = 'cypress/screenshots/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js';
```
Comment (ActiveChooN) [inline tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js:12]: It's better to not use file extension in folder name.

File: tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                `${screenshotsPath}/canvas3d_perspective_after_add_cuboid.png`,
+            );
+        });
+
+        it('Cuboid interaction by mouse.', () => {
+            // The cuboid does not have time to change color so we make the mouse movement 3 times.
+            cy.get('.cvat-canvas3d-perspective')
+                .trigger('mousemove', 500, 200)
+                .trigger('mousemove', 400, 200)
+                .trigger('mousemove', 300, 200); // The cuboid should change a color after movement cursor from it
```
Comment (ActiveChooN) [inline tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js:40]: Just a question: why exactly three movement is used here and further, not two or simple delay? Is it not working?

Comment (bsekachev) [reply tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js:40]: @ActiveChooN 

Delays lead to indeterminate (when sometimes tests passed, sometimes tests failed). It is better to try escaping them if possible.

File: tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js:12]: In this form, cypress creates directories for placing screenshots.

File: tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js:40]: Yes, this is done to exclude the use of wait(). The color of the cuboid does not change immediately when the mouse interacts with it.

**Review by ActiveChooN — APPROVED** [review]

**Pull request merged**